.golangci.yaml

```diff
@@ -0,0 +1,14 @@
+linters:
+  # initially we disable all linters, since this is an old code base with quite a lot of issues
+  disable-all: true
+  enable:
+  - gocyclo
+#  - staticcheck
```

Comment:
Is there a plan to enable these over time? Should we open an issue to keep track of this?

Reply:
Makes sense: https://github.com/kubevirt/project-infra/issues/3837